configure/CMakeLists.txt

```diff
@@ -1,23 +1,37 @@
 include(FindPkgConfig)
 
 if(NOT CMAKE_BUILD_TYPE)
-    message("No build type specified - default is DEBUG")
-    set(CMAKE_BUILD_TYPE DEBUG)
+    message("No build type specified - default is RELEASE")
+    set(CMAKE_BUILD_TYPE RELEASE)
```

Comment:
I don't mind changing the default build type but I think we should rather use Release and Debug words everywhere instead of RELEASE and DEBUG because they are the standard ones as described on CMAKE_BUILD_TYPE documentation (https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html).
We have to be aware that STREQUAL and MATCHES are case sensitive in CMake.
If we agree on doing this change, we will need to update line 148 from configure/CMakeLists.txt too and replace
`if(CMAKE_BUILD_TYPE MATCHES RELEASE)`
with
`if(CMAKE_BUILD_TYPE MATCHES Release)`
I think we should also update the README.md and INSTALL.md files to be consistent (Replace RELEASE|DEBUG with Release|Debug in the CMake command example.

Reply:
No problem for me, I will need to update more files since I'm using this "if" in some places